get-armory-cli

```diff
@@ -25,6 +36,23 @@ case $RAW_ARCH in
       ;;
 esac
 
+# Sanity checks
+
+echo "Looking for dependencies..."
+if ! command -v curl > /dev/null; then
+	echo "######################################################################################################"
+	echo "# curl not found."
+	echo "#"
+	echo "# Please install curl on your system using your favourite package manager."
```

Comment:
```suggestion
	echo "# Please install curl on your system using your favorite package manager."
```

"favourite" is British 🤷

Reply:
"then re-run this script."

Reply:
> "favourite" is British 🤷

I'm so pro in english that I can't distinguish which one to use british or american